src/main.rs

```diff
@@ -22,8 +22,8 @@ async fn main() -> Result<()> {
 
         let app = App::new(config.terminal.maximum_messages as usize);
 
-        let (twitch_tx, terminal_rx) = mpsc::channel(1);
-        let (terminal_tx, twitch_rx) = mpsc::channel(1);
+        let (twitch_tx, terminal_rx) = mpsc::channel(100);
+        let (terminal_tx, twitch_rx) = mpsc::channel(100);
```

Comment:
Why are we increasing this to 100?

Reply:
100 is arbitrary, anything above 1 is sensible. A channel buffer of 1 will make `.send`s sleep until the message is removed by the receiver. Thus, it doesn't make sense to only have one message in queue.

Reply:
I see. Thank you for the confirmation.